### ISSUE TITLE
How to utilize/monitor per-process metrics

### ISSUE THREAD

**Issue by alfieyfc** [yabeda-rb/yabeda-sidekiq#26]
We're interested in taking a look at the [local metrics](https://github.com/yabeda-rb/yabeda-sidekiq#local-per-process-metrics) (ie. `sidekiq_running_job_runtime `, `sidekiq_job_runtime`, etc.) but can't find how to do so from the docs and conversations from previous issues/PRs. 

I tried googling too but didn't find much relevant resource either. Is what we're trying to do possible? Any directions or references would be very much appreciated! :) 🙏

**Comment by Envek** [comment]
What monitoring system are you going to use?

In yabeda, you take a [_plugin_ to collect metrics](https://github.com/yabeda-rb/yabeda#available-plugins-to-collect-metrics) (yabeda-sidekiq is a plugin), and then you take an [_adapter_ for monitoring system](https://github.com/yabeda-rb/yabeda#available-monitoring-system-adapters) (e.g. [yabeda-prometheus](https://github.com/yabeda-rb/yabeda-prometheus) for Prometheus) and set up exporting metrics from every Sidekiq worker process to monitoring. And you can write your own adapter to send metrics wherever you want (e.g. built-in [`Yabeda::TestAdapter`](https://github.com/yabeda-rb/yabeda/blob/master/lib/yabeda/test_adapter.rb) just stores values in memory).

Please read introductory blog post here: [Meet Yabeda: A Ruby instrumentation framework](https://evilmartians.com/chronicles/meet-yabeda-modular-framework-for-instrumenting-ruby-applications) and then please ask more detailed questions: what do you want to achieve?

**Comment by alfieyfc** [comment]
We're using `yabeda-prometheus` on Grafana. I just checked again and the metrics are there. It probably just didn't load up properly somehow when I filed this issue. 😓

<img width="458" alt="image" src="https://user-images.githubusercontent.com/41598103/167589597-d5623189-2882-408c-8fad-d01c596708a9.png">

Thanks for the blog post reference, though. It'll be very handy as I'm setting up the dashboards now. Thank you!
